src/openrct2/network/Network.cpp

```diff
@@ -31,7 +31,7 @@
 // This string specifies which version of network stream current build uses.
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
-#define NETWORK_STREAM_VERSION "2"
+#define NETWORK_STREAM_VERSION "3"
```

Comment:
Unnecessary, please revert this change. This only needs changing when the change could cause two clients to run different logic and lead to them inferring different state from same input.

Reply:
Alright! thx for the info! btw is there any resources I can read to get a better understanding of how everything works,apart from the source code of course.